cesm2_albedo/calculate_bareground_alb.jl

```diff
@@ -0,0 +1,51 @@
+using NCDatasets
+using Dates
+using Statistics
+
+
+ds = NCDataset("sw_albedo_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412_v2_no-nans.nc")
+
+# convert "no leap" dates to standard
+dates =
+    Dates.DateTime.(
+        reinterpret.(Ref(NCDatasets.DateTimeStandard), ds["time"][:])
+    )
+latitude = ds["lat"][:]
```

Comment:
I'm surprised we can still use this syntax - I thought it got replaced with `Array(ds["lat"])`, but maybe that's for variables not dimensions? No need to change it, just something I noticed